USAGE

```diff
@@ -92,6 +87,14 @@ OPTIONAL ARGUMENTS (current value)
 --dwi_resolution                            DWI resolution ($dwi_resolution).
 --dwi_interpolation                         DWI interpolation method [nn, lin, quad, cubic] ($dwi_interpolation).
 
+--max_dti_shell_value                       Maximum shell threshold to be consider as a DTI shell ($max_dti_shell_value).
```

Comment:
Maybe you should state here that max_dti_shell_value is used by default, and dti_shells will overide max_dti_shell_value.  Not necessary but the multiple options might confuse a user